src/test/java/com/alibaba/wasp/jdbc/TestJdbcStatement.java

```diff
@@ -61,13 +61,16 @@ public static void beforeClass() throws Exception {
 
   @AfterClass
   public static void afterClass() throws Exception {
-    conn.close();
-    TEST_UTIL.shutdownMiniCluster();
+    if (conn!=(Connection)null){
+      conn.close();
+      TEST_UTIL.shutdownMiniCluster();
+    }
   }
 
   @Test
   public void testStatement() throws SQLException, IOException,
       InterruptedException {
+    if (conn!=(Connection)null){
```

Comment:
Instead of this if statement, you may use the following instead.
https://junit.org/junit4/javadoc/4.12/org/junit/Assume.html#assumeNotNull(java.lang.Object...)

Reply:
Made the change.